CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## 2.5.0
+
+[Full Changelog](https://github.com/cloudfoundry-incubator/stratos/compare/2.4.0...2.5.0)
+
+This release contains a number of fixes and improvements:
+
+**Fixes:**
+
+- Fix persist of dashboard settings [\#3730](https://github.com/cloudfoundry-incubator/stratos/pull/3730)
```

Comment:
This one's an entity catalogue/v3-master one

Reply:
Ah, okay